indexer/config/config_test.go

```diff
@@ -257,3 +257,49 @@ func TestLocalDevnet(t *testing.T) {
 
 	require.Equal(t, devnetPreset.ChainConfig.L1Contracts, conf.Chain.L1Contracts)
 }
+
+func TestThrowsOnUnknownKeys(t *testing.T) {
```

Comment:
Nit go doesn't throw
```suggestion
func TestErrOnUnknownKeys(t *testing.T) {
```